OutSystems/outsystems-ui
# ROU-4570: Datepicker/DatepickerRange doesn't get focused when is opened via API

This PR is to fix issue on Datepicker Open API to focus on input element when is triggered

### What was happening
- When the Datepicker is opened using the API, the focus are the button clicked
<img width="964" alt="Screenshot 2023-10-02 at 11 36 01" src="https://github.com/OutSystems/outsystems-ui/assets/25321845/758b5304-13d3-4ced-aeda-757a58f1ffbe">

### What was done
- The Datepicker open API was reviewed to focus the input element when is triggered, to keep the same behavior as the default when is opened by input click
![image](https://github.com/OutSystems/outsystems-ui/assets/25321845/e1c7d49d-1ee2-4b65-99ec-d41853c398f6)


### Test Steps

1. Go to each sample page
2. Enable the A11Y switch
3. Click on apply button
4. Open the datepicker by the button on right top of input
5. Expected: the datepicker will open and the input will be focused

### Checklist
